python/docs/contents.rst

```diff
@@ -1,44 +1,16 @@
 .. code-block:: bash
```

Comment:
I thnk this can be deleted?

Reply:
true. removed it. I also realized it didn't generate the docs for all the sub-commands 🤔 . I will look this next via https://github.com/projectnessie/nessie/issues/2614

Reply:
nice catch :)